src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImage.groovy

```diff
@@ -64,34 +64,58 @@ class DockerBuildImage extends AbstractDockerRemoteApiTask implements RegistryCr
     @Optional
     final SetProperty<String> tags = project.objects.setProperty(String)
 
+    /**
+     * When <code>true</code>, do not use docker cache when building the image.
```

Comment:
The option would be to use `{@code}`. Would you mind using that instead?

Reply:
Done